index.html

```diff
@@ -653,6 +651,70 @@ <h1>Basic Concepts</h1>
 toward the end of the document.
       </p>
 
+      <section>
+        <h2>Contexts</h2>
+
+        <p>
+When two software systems need to exchange data, they must use terminology
+that both systems understand. A useful analogy is to think about how two
+people communicate; both people must use the same language and the words
+that each of them use must mean the same thing to the other party. This
+specification uses the <code>@context</code> property to express the context
+of the conversation.
+        </p>
+
+        <dl>
+          <dt><dfn>@context</dfn></dt>
+          <dd>
+The value of this property MUST be one or more URIs where first URI is the
```

Comment:
The meaning of a verifiable credential could change if its context changes. I'm concerned that providing only a URI (which by its nature is a mutable reference), we are not requiring any sort of guarantee that the context pointed to by the URI won't change over time.

Reply:
This is also my concern, especially since it is proposed to move Issuer Terms of Use to the @context from the VC itself. The advantage of this move is that the holder of a ZPK credentials cannot hide the terms of use, which he could if it were an assertion in the VC. The disadvantage is that the verifier has to rely on the contents of URLs being immutable. Can we propose some crypto to guarantee that if the contents change, the verifier can detect this.

Reply:
We can't solve this issue in this PR. As @gkellogg mentioned, it's a separate broader Web/JSON-LD ecosystem issue. There are options, but we are unlikely to settle on a standard before the VCWG is done. I know that @dmitriz is going to document all the options over in the JSON-LD WG's issue repository.